### ISSUE TITLE
Deploying software under test (SUT) outside Docker does not work

### ISSUE THREAD

**Issue by pjakovits** [radon-h2020/radon-ctt#79]
When trying to test software which does not use Docker, the CTT server throws an error when trying to look up the IP/Hostnames of the deployed services. It seems that it specifically looks up for the hostname using Docker CLI commands. 

I wonder if it would be better to try to get the IP address or hostname of the deployed SUT from the orchestrator. For example, when using OpenStack to deploy instances, IP addresses of the deployed instances are available in the instance model and also saved in .opera folder. 

SUT for our case is the thumbnail generator data pipeline for reading files from an AWS S3 bucket and trigger an AWS Lambda function whenever a new image is uploaded into the S3 bucket. The data pipeline is deployed on top of an Apache Nifi node, hosted as an instance in OpenStack.

For additional information, the github project we used is available here: https://github.com/Afsana2910/demo-ctt-thumbgen
Services (SUT and TI) are inside the radon-ctt foder as CSAR files, just like in the sockshop example.

**Comment by duelle** [comment]
Thanks for reporting this issue. 
Did you set `-e CTT_FAAS_ENABLED="1"` in the `docker run` command for CTT and if not, could you please try it with that flag set?

**Comment by pjakovits** [comment]
We tried it with `-e CTT_FAAS_ENABLED="1"` but it still initially gave the same error.

We are not testing FaaS, but rather we aim to test a Data pipeline deployed in OpenStack by using JMeter to uploading images to an S3 bucker using a Jmeter test plan. 

We got it to work now by by modifying the line 120  in:
https://github.com/radon-h2020/radon-ctt/blob/master/ctt-server/models/deployment.py
from
`docker_network = subprocess.getoutput("docker network ls | grep compose | head -n1 | awk '{print $2}'")`
to 
`docker_network = subprocess.getoutput("docker network ls | grep bridge | head -n1 | awk '{print $2}'")`

It seems CTT Server is expecting a docker network with name docker_compose to exist, which did not exist in our server.
